web/packages/teleport/src/services/auth/auth.ts

```diff
@@ -16,7 +16,11 @@ limitations under the License.
 
 import api from 'teleport/services/api';
 import cfg from 'teleport/config';
-import { DeviceType, DeviceUsage } from 'teleport/services/mfa';
+import {
+  DeviceType,
+  DeviceUsage,
+  IsMfaRequiredRequest,
```

Comment:
i'd move `IsMfaRequiredRequest` typings to auth/types.ts, since the check is done in auth.ts